src/api/app/views/webui2/webui/project/_project_inherited_packages.html.haml

```diff
@@ -0,0 +1,14 @@
+.card-body
+  - if ipackages.present?
+    %table.table.table-striped.table-bordered#inherited-packages-table
+      %thead
+        %tr
+          %th Name
+          %th From
+      %tbody
+        - ipackages.each do |ipackage|
+          %tr
+            %td
```

Comment:
I think this column should take more space (as some packages have long names). Perhaps 75% of the space available in the table.